### PR TITLE
Ruby and Docker

### PR DESCRIPTION
**Pull request by bibendi**
### What is the purpose of this pull request?

There is a code that doesn't compatible with Ruby 2.2.

### What changes did you make? (overview)

- [x] Added required_ruby_version to gemspec and rubocop
- [x] Added Docker configs

### CODE REVIEW

File: anycable-rails.gemspec [hunk 1]
```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files README.md MIT-LICENSE CHANGELOG.md lib`.split
   spec.require_paths = ["lib"]
 
+  spec.required_ruby_version = ">= 2.3"
```
Comment (sponomarev) [inline anycable-rails.gemspec]: `Ruby >= 2.4` is mentioned in the docs

Comment (palkan) [reply anycable-rails.gemspec]: Yeah, `anycable` requires 2.4+, so should we do here

https://github.com/anycable/anycable/blob/b1d0fe0cd125fe3ea02dcd04059d6a30ae63a918/anycable.gemspec#L29

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+version: '2'
```
[the remark this answers is not in the document]
Comment (bibendi) [reply docker-compose.yml]: Yep, better using the third version

**Comment by bibendi** [comment]
Fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+- Require a minimum version of Ruby when installing the gem. ([@bibendi][])
```
Comment (palkan) [inline CHANGELOG.md]: Please, add shortcut definition to the end of the file (currently, `@bibendi` link leads nowhere).

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
...
+
+- Add ability developing the gem in Docker. ([@bibendi][])
```
Comment (palkan) [inline CHANGELOG.md]:
```suggestion
- Add ability to develop the gem with Docker. ([@bibendi][])
```

**Pull request merged**